extension/src/popup.js

```diff
@@ -94,66 +110,99 @@ stopToonin.onclick = () => {
     });
 }
 
+createRoomBtn.onclick= () => {
+    createRoomBtn.style.display = "none";
+    connectRoomBtn.style.display = "none";
+    shareButton.style.display = "block";
+    roomNameSpan.style.display = "block";
+    btnBack.style.display = "block";
+
+}
+
+connectRoomBtn.onclick= () => {
+    createRoomBtn.style.display = "none";
+    connectRoomBtn.style.display = "none";
+    roomNameToonin.style.display = "block";
+    btnBack.style.display = "block";
+    playButton.style.display = "block";
+    connectSpan.style.display = "inline-flex";
+
+}
+
 chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
     var activeTab = tabs[0];
     chrome.tabs.sendMessage(activeTab.id, {"message": "clicked_extension"});
 });
 
   chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
     if (request.message === "extension_state_from_background" && request.data.roomID) {
-        roomNameSpan.style.visibility= "hidden";
-        shareButton.style.visibility= "hidden";
-        stopSharingButton.style.visibility="visible";
-        copyButton.style.visibility="visible";
-        playButton.style.visibility = "hidden";
-        stopToonin.style.visibility = "hidden";
-        connectSpan.style.visibility = "hidden";
+        roomNameSpan.style.display = "none";
+        shareButton.style.display = "none";
+        stopSharingButton.style.display = "block";
+        copyButton.style.display = "block";
+        playButton.style.display = "none";
+        stopToonin.style.display = "none";
+        connectSpan.style.display = "none";
         roomID=request.data.roomID;
-        muteSpan.style.visibility = "visible";
+        muteSpan.style.display = "block";
         muteBtn.checked = request.data.muted;
         muteStatus.hidden = !muteBtn.checked;
         sessionIDText.innerHTML = "Your Toonin ID is: \n" + roomID;
-        sessionIDText.style.visibility = "visible";
-        peerCounter.style.visibility = "visible";
+        roomDiv.style.display = "block";
+        peerCounter.style.display = "block";
         peerCounter.innerHTML = "You have " + request.data.peerCounter + " listeners.";
-    } 
+        roomNameSpan.style.display = "none";
+        btnShare.style.display = "none";
+        backButton.style.display = "none";
+    }
     else if (request.message === "extension_state_from_background" && !request.data.roomID && request.data.playing) {
```

Comment:
call to function connectRoomBtn (line 122) can be made and disable the create room functionalities.